source/style-guide.txt

```diff
@@ -64,11 +61,11 @@ MongoDB external and internal customers.
 The |mdbsg| includes material from the following:
 
 - `Google Developer Documentation Style Guide <https://developers.google.com/style/>`__
-  by `Google <https://www.google.com>`__, licensed under
-  :cc:`CC BY 4.0 </by/4.0/>`;
+  by `Google <https://www.google.com>`__, licensed under `CC BY 4.0
```

Comment:
Should this also use double-underscore or was a single one intentional? I think when using a single one, [you can separate the definition and avoid repetition](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages), but I don't think we use that pattern often.

Reply:
Another good catch!